config.yaml

```diff
@@ -44,7 +44,7 @@ options:
   auth-provider-url:
     description: |
         The .well-known IDP discovery URL for authentication and authorization.
-    default: ""
+    default: "https://accounts.google.com"
```

Comment:
This now doesn't match the default value in the Jinja, could be worth updating to match or removing the default from the jinja all together.